### DISCUSSION TITLE
Custom Reports - Support for smb shares

### DISCUSSION THREAD

**Discussion by AGlennDa** [trimble-oss/dba-dash#1126] [Ideas]
I have created a report showing the latest backup location. 
The location is on fileshare (smb). 
With the recent feature to make a column URL friendly, I was thinking it would be nice to add a column that is SMB friendly. 
So when you click on a smb link (e.g., smb:\\some_share\backups\) it would then open File Explorer, and browse the path. 
I have tried simply adding it, and making it a URL, but without luck. 
It simply prompts with "Invalid URL: \\path_to_share\backup".

**Comment by DavidWiseman** [comment]
This will be included in the next release.  🎉

If you want to try it out now, there is a custom build [here](https://github.com/DavidWiseman/dba-dash/releases/tag/3.14.0) with the fix included.

**Reply by AGlennDa** [reply]
Nice! :) Thank you.
